python/pyspark/sql/types.py

```diff
@@ -1730,7 +1730,28 @@ def _check_series_convert_timestamps_internal(s, timezone):
     # TODO: handle nested timestamps, such as ArrayType(TimestampType())?
     if is_datetime64_dtype(s.dtype):
         tz = timezone or 'tzlocal()'
-        return s.dt.tz_localize(tz).dt.tz_convert('UTC')
+        """
+        tz_localize with ambiguous=False has the same behavior of pytz.localize
```

Comment:
I'm not sure we want this doctest

Reply:
Oh definitely not doctest..Let me change to comments